test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -271,11 +282,11 @@ void UcxContext::progress_conn_requests()
 
 void UcxContext::progress_io_message()
 {
-    if (!_iomsg_recv_request->completed) {
+    if (_use_am || !_iomsg_recv_request->completed) {
```

Comment:
do we really have to check _use_am here

Reply:
_iomsg_recv_request is initialized with `NULL`

Reply:
ok